### ISSUE TITLE
When importing from csv, no formatting

### ISSUE THREAD

**Issue by brandonopened** [opensalt/opensalt#253]
We need to be able to specify paragraph breaks from an imported csv file. Due to html sanitizing, cannot do so currently

**Comment by brandonopened** [comment]
Have tried inserting <br> (broke csv), tried two spaces, no effect on output

**Comment by roverwolf** [comment]
Note that there is no "html sanitizing" happening, the fullstatement is treated as markdown (with no HTML allowed) and then converted to be presented with HTML.

**Comment by brandonopened** [comment]
Ah, I was trying the formatting in “notes” I assume same treatment? Thanks for clarifying

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by roverwolf** [comment]
Yes, the "notes" field is treated the same way as the full statement field.

**Comment by brandonopened** [comment]
within the importer: straight import
within markdown (rendering) : allow some HTML tags with sanitization.

**Comment by brandonopened** [comment]
proposal allowed: 
<li>
<u>
<i>
<b>
<p>

**Comment by brandonopened** [comment]
Question: So do we want to allow a subset of HTML tags... or allow sanitized HTML in general.

**Comment by roverwolf** [comment]
@brandonopened Is this closable now with the limited support of HTML we have now?
